### ISSUE TITLE
a way to leave differently sized empty spaces above and between images

### ISSUE THREAD

**Issue by mplusb** [MachoThemes/modula-lite#276]
<img width="1440" alt="Screen-Shot-2020-01-24-at-5-37-35-PM" src="https://user-images.githubusercontent.com/26008574/73251349-76739000-41c1-11ea-9b0c-66952ab0f7d7.png">

**Comment by cristianraiber** [comment]
This could be easily addressed once we move onto "custom blocks" into the grid and add a "spacer" block;

**Comment by ElectricFeet** [comment]
Love this idea!

They would need to collapse responsively on tablet/mobile. @mplusb How would the layout above collapse responsively?

**Comment by cristianraiber** [comment]
@ElectricFeet - I think we need a secon opinion on this from @giucu91 - but I think the easiest way to handle this would be column-based, where, on mobile, you'd have to loose this type of layout as there's not enough space to have something like this :)

**Comment by ElectricFeet** [comment]
Yes: on mobile, it would be all collapsed down. So you'd just not show the spacers that you inserted.

On tablets, however, it's more up for debate. In the layout above, for example, on a tablet you might want to go down from four columns to 2 but keep the spacers, to give the images some fluidity. 

Also collapsing to columns (rather than what could loosely be called "rows") in the example above would separate the poet from his bio, so may not give what the user intended.

**Comment by ElectricFeet** [comment]
> I for one would love to see the possibility of collapsing by "rows" -- that is, the next-highest image on the grid goes next (& iterate). That way, you broadly keep the "order" of images as the viewer scrolls down the page, whether on desktop, tablet, or phone.

Ah: thinking about this further. You could add a pro feature on sorting that allows us to assign a number to each image, allowing sorting however we like.

I'll open a different issue for this ordering enhancement in Pro.

Back to the spacing theme of this thread: you'll need to decide what to do on tablets, where things could get confusing with the spacers.
